static/app/views/app/index.spec.tsx

```diff
@@ -257,27 +255,4 @@ describe('App', function () {
       ]);
     });
   });
-
-  it('sets theme property for chonk-ui depending on feature flag', async function () {
-    function Component() {
-      const theme = useTheme();
-      return <div>isChonk: {String(theme.isChonk)}</div>;
-    }
-
-    render(
-      <App {...routerProps}>
-        <Component />
-      </App>
-    );
-
-    expect(await screen.findByText('isChonk: false')).toBeInTheDocument();
-
-    const chonkOrganization = OrganizationFixture({features: ['chonk-ui']});
-
-    act(() => {
-      OrganizationStore.onUpdate(chonkOrganization, {replace: true});
-    });
-
-    expect(await screen.findByText('isChonk: true')).toBeInTheDocument();
-  });
```

Comment:
The removal here is taken from this PR https://github.com/getsentry/sentry/pull/85389